spec/features/staff/teammates_spec.rb

```diff
@@ -88,4 +88,35 @@
     end
   end
 
+  context "Organizer can't send invitations if contact email for event isn't set" do
```

Comment:
I just left a comment in another PR where your use of context is different than what I'd expect. I'm not 100%, so we could probably just chat about it and see what makes the most sense here. I think your intent is pretty clear, so it's not a huge deal.